src/webui/www/private/scripts/contextmenu.js

```diff
@@ -746,15 +728,12 @@ window.qBittorrent.ContextMenu ??= (() => {
                     break;
             }
         }
-    });
+    };
 
-    const RssArticleContextMenu = new Class({
-        Extends: ContextMenu
-    });
+    class RssArticleContextMenu extends ContextMenu {};
```

Comment:
```suggestion
    class RssArticleContextMenu extends ContextMenu {
    };
```

Reply:
Formatter makes it into one line. It only works if we add an empty line between curly braces - should I do it or keep it like it is?

Reply:
>Formatter makes it into one line. 

Ok, forget it. We can keep it as is.

BTW, I would appreciate when you amend commits don't do a rebase on master. Github provides this link to show the diff between your push. For example, ` skomerko force-pushed the webui-context-class branch from ...` you can click on the `force-pushed` text to see the diff. And if you do a rebase, it will include many other unrelated changes.